nuget/FSharp.Data.nuspec

```diff
@@ -16,43 +16,40 @@
-    <file src="..\bin\FSharp.Data.DesignTime.pdb" target="lib/net45" />
-    <file src="..\bin\FSharp.Data.DesignTime.xml" target="lib/net45" />
-    <file src="..\bin\portable7\FSharp.Data.dll" target="lib/portable-net45+netcore45" />
-    <file src="..\bin\portable7\FSharp.Data.pdb" target="lib/portable-net45+netcore45" />
-    <file src="..\bin\portable7\FSharp.Data.xml" target="lib/portable-net45+netcore45" />
-    <file src="..\bin\portable7\FSharp.Data.DesignTime.dll" target="lib/portable-net45+netcore45" />
-    <file src="..\bin\portable7\FSharp.Data.DesignTime.pdb" target="lib/portable-net45+netcore45" />
-    <file src="..\bin\portable7\FSharp.Data.DesignTime.xml" target="lib/portable-net45+netcore45" />
-    <file src="..\bin\portable259\FSharp.Data.dll" target="lib/portable-net45+netcore45+wpa81+wp8" />
-    <file src="..\bin\portable259\FSharp.Data.pdb" target="lib/portable-net45+netcore45+wpa81+wp8" />
-    <file src="..\bin\portable259\FSharp.Data.xml" target="lib/portable-net45+netcore45+wpa81+wp8" />
-    <file src="..\bin\portable259\FSharp.Data.DesignTime.dll" target="lib/portable-net45+netcore45+wpa81+wp8" />
-    <file src="..\bin\portable259\FSharp.Data.DesignTime.pdb" target="lib/portable-net45+netcore45+wpa81+wp8" />
-    <file src="..\bin\portable259\FSharp.Data.DesignTime.xml" target="lib/portable-net45+netcore45+wpa81+wp8" />
+    <!-- Place the TPRTCs in the usual locations --> 
+    <file src="..\src\FSharp.Data\bin\Release\net45\FSharp.Data.dll" target="lib/net45" />
+    <file src="..\src\FSharp.Data\bin\Release\net45\FSharp.Data.pdb" target="lib/net45" />
+    <file src="..\src\FSharp.Data\bin\Release\net45\FSharp.Data.xml" target="lib/net45" />
+    <file src="..\src\FSharp.Data\bin\Release\netstandard2.0\FSharp.Data.dll" target="lib/netstandard2.0" />
+    <file src="..\src\FSharp.Data\bin\Release\netstandard2.0\FSharp.Data.pdb" target="lib/netstandard2.0" />
+    <file src="..\src\FSharp.Data\bin\Release\netstandard2.0\FSharp.Data.xml" target="lib/netstandard2.0" />
+
+    <!-- Place the TPDTC in the legacy location alongside FSharp.Data --> 
+    <file src="..\src\FSharp.Data.DesignTime\bin\Release\net45\FSharp.Data.DesignTime.dll" target="lib/net45" />
+    <file src="..\src\FSharp.Data.DesignTime\bin\Release\net45\FSharp.Data.DesignTime.pdb" target="lib/net45" />
+
+    <!-- Place the TPDTC in the future-proof location --> 
+    <file src="..\src\FSharp.Data.DesignTime\bin\Release\net45\FSharp.Data.DesignTime.dll" target="typeproviders/fsharp41/net45" />
+    <file src="..\src\FSharp.Data.DesignTime\bin\Release\net45\FSharp.Data.DesignTime.pdb" target="typeproviders/fsharp41/net45" />
+    <file src="..\src\FSharp.Data.DesignTime\bin\Release\netstandard2.0\FSharp.Data.DesignTime.dll" target="typeproviders/fsharp41/netstandard2.0" />
+    <file src="..\src\FSharp.Data.DesignTime\bin\Release\netstandard2.0\FSharp.Data.DesignTime.pdb" target="typeproviders/fsharp41/netstandard2.0" />
```

Comment:
I'm reading a bunch of stuff about this, but still didn't really understand why the design time dll going forward cannot be in the same dir as the runtime dll but has to be on a typeproviders subdirectory